templates/base.jinja2

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name='discription' content="Leaning Journal at CodeFellows"> 
+    <title>Saki's Learning Journal</title>
+    <!--[if lt IE 9]>
+    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+
+    <!-- ADD THE FOLLOWING LINE ONLY -->
+    <link href="/static/style.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <header>
+      <div id='title-1'>
+            <img src="static/nmr_2.png" class="img" width='100px' height='70px'>
```

Comment:
I think I forgot to remove unnecessary codes, which I put when I was trying
to figure things out. I'll remove them.


On Mon, Jun 29, 2015 at 9:39 PM, Cris Ewing <notifications@github.com>
wrote:

> In templates/base.jinja2
> <https://github.com/SakiFu/learning-journal/pull/5#discussion_r33539957>:
>
> > +<html lang="en">
> > +  <head>
> > +    <meta charset="utf-8">
> > +    <meta name='discription' content="Leaning Journal at CodeFellows">
> > +    <title>Saki's Learning Journal</title>
> > +    <!--[if lt IE 9]>
> > +    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
> > +    <![endif]-->
> > +
> > +    <!-- ADD THE FOLLOWING LINE ONLY -->
> > +    <link href="/static/style.css" rel="stylesheet" type="text/css">
> > +  </head>
> > +  <body>
> > +    <header>
> > +      <div id='title-1'>
> > +            <img src="static/nmr_2.png" class="img" width='100px' height='70px'>
>
> good use of the static view to get at your (*super* creepy) logo image :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SakiFu/learning-journal/pull/5/files#r33539957>.
>